src/sidecar/middlewares.ts

```diff
@@ -106,6 +106,18 @@ export class ErrorResponseMiddleware implements Middleware {
         return;
       }
 
+      // Special case: suppress error logs for schema registry RBAC-related 404s
+      // Handles both gateway format and direct SR endpoint format, including versions and versions/latest
+      if (
+        context.response.status === 404 &&
+        /(gateway\/v1\/schema-registry\/[^/]+\/subjects\/[^/]+\/versions|:\d+\/subjects\/[^/]+\/versions)/.test(
```

Comment:
```suggestion
        /\/subjects\/[\w-]/.test(
```
We don't see `gateway/v1/schema-registry/...` as part of `context.response.url` though, so this is probably all we need to check (`/subjects/{subject}` to include alphanumerics, underscores, and dashes)